### DISCUSSION TITLE
Anti-Spam IRC for make less "Excess Flood"

### DISCUSSION THREAD

**Discussion by mrc4tt** [piqueserver/piqueserver#698] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Could be nice, if we/or could possibly to get/or make some function for IRC bot., so it has "Anti Spam" for making less **Excess Flood**

**Describe the solution you'd like**
Make some anti-spam for BOT (not getting **Excess Flood**)

**Describe alternatives you've considered**
?? - Dunno how it should be possible to get make.

**Additional context**
When, someone trying to spam on piqueserver, and the bot will be "killed" from IRC because of **Excess Flood**

**Comment by NotAFile** [comment]
So basically the complaint is that spam in chat will have the bot kicked fot flooding.

I wonder how this occurs, since piqueserver is set to mute anyone who writes more than 5 messages at a rate of more than one message every two seconds.

Perhaps this is broken?

**Comment by mrc4tt** [comment]
Nvm, I did fix it with creating own IRC Server. without using quakenet! =)

**Comment by NotAFile** [comment]
I still think this is an issue in general, so I'll reopen this. Feel free to unsubscribe.
